components/centraldashboard/app/api.ts

```diff
@@ -40,6 +55,20 @@ export class Api {
     return email;
   }
 
+  /**
+   * Retrieves workgroup info from Profile Controller.
+   */
+  private async getWorkgroup(req: express.Request, user: string): Promise<WorkgroupInfo> {
+    const [profileController, auth] = this.getProfileAndAuth(req);
+    const adminResponse = await profileController.v1RoleClusteradminGet(user, auth);
+    const bindings = await profileController.readBindings(user, auth);
```

Comment:
It seems like the readBindings method expects a string that is only the user's email address here and thus fails to compile.

Reply:
Fixed, thanks!